Apps/Examples/Examples/All Examples/SymbolClusteringExample.swift

```diff
@@ -0,0 +1,166 @@
+import UIKit
+import MapboxMaps
+
+@objc(SymbolClusteringExample)
+
+public class SymbolClusteringExample: UIViewController, ExampleProtocol {
+
+    internal var mapView: MapView!
+
+    override public func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Create a map view centered over Washington, DC.
+        let center = CLLocationCoordinate2D(latitude: 38.889215, longitude: -77.039354)
+        let cameraOptions = CameraOptions(center: center, zoom: 11)
+        let mapInitOptions = MapInitOptions(cameraOptions: cameraOptions, styleURI: .dark)
+        mapView = MapView(frame: view.bounds, mapInitOptions: mapInitOptions)
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+
+        view.addSubview(mapView)
+
+        // Add the source and style layers once the
+        mapView.mapboxMap.onNext(.mapLoaded) { _ in
+            self.addSymbolClusteringLayers()
+        }
+        
+        let tap = UITapGestureRecognizer(target: self, action: #selector(handleTap(gestureRecognizer:)))
+        mapView.addGestureRecognizer(tap)
+    }
+
+    func addSymbolClusteringLayers() {
+        let style = self.mapView.mapboxMap.style
+        // The image named `fire-station-11` is included in the app's Assets.xcassets bundle.
+        let image = UIImage(named: "fire-station-11")!.withRenderingMode(.alwaysTemplate)
```

Comment:
Rendering modes can also be configured in the asset catalogue (Assets.xcassets). If you do that, you could change this line to:

```suggestion
        let image = UIImage(named: "fire-station-11")!
```

Reply:
Agreed. I did it in the code to highlight that it should be done, but will add a comment to explain that it can be done in the asset catalogue.

Reply:
Looks like it's already configured that way. Does the example work if you make this change?